pol-core/pol/polcfg.cpp

```diff
@@ -78,12 +78,10 @@ void PolConfig::read_pol_config( bool initial_load )
     Plib::systemstate.config.web_server = elem.remove_bool( "WebServer", false );
     Plib::systemstate.config.web_server_port = elem.remove_ushort( "WebServerPort", 8080 );
 
-    unsigned short max_tile = elem.remove_ushort( "MaxTileID", UOBJ_DEFAULT_MAX );
-
-    if ( max_tile != UOBJ_DEFAULT_MAX && max_tile != UOBJ_SA_MAX && max_tile != UOBJ_HSA_MAX )
-      Plib::systemstate.config.max_tile_id = UOBJ_DEFAULT_MAX;
-    else
-      Plib::systemstate.config.max_tile_id = max_tile;
+    if ( elem.has_prop( "MaxTileID" ) )
+    {
+      INFO_PRINT << "Warning: MaxTileID has been removed from pol.cfg.\n";
```

Comment:
Testsuite has a pol.cfg which contains the setting and we have a Pol.cfg.example

Reply:
MaxTileID is now only used in POL to further restrict what is contained in tiles.cfg. Docs and configs were updated to represent that